pkg/migrations/sql/idempotent/telemetry.sql

```diff
@@ -0,0 +1,94 @@
+-- promscale_telemetry_housekeeping() does telemetry housekeeping stuff, which includes
+-- searching the table once an hour, looking for stale promscales, if found,
+-- adding their values into the counter_reset row, and then cleaning up the stale
+-- promscale instances.
+-- It is concurrency safe, since it takes lock on the promscale_instance_information table,
+-- making sure that at one time, only one housekeeping is being done.
+-- 
+-- It returns TRUE if last run was beyond an hour, otherwise FALSE.
+CREATE OR REPLACE FUNCTION SCHEMA_PS_CATALOG.promscale_telemetry_housekeeping()
+RETURNS BOOLEAN AS
```

Comment:
Instead of hardcoding the intervals (1 hour, 2 hours), I would make them parameters to this function. The parameters can have default values of 1 and 2 hours. Having them as parameters will make it easier to test and easier to make it configurable should we choose to.